scripts/measure-testgrid-flakiness.sh

```diff
@@ -10,6 +10,12 @@ then
 fi
 
 pushd ./tools/testgrid-analysis
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-e2e-amd64
-go run main.go flaky --create-issue --dashboard=sig-etcd-periodics --tab=ci-etcd-unit-test-amd64
+# ci-etcd-e2e-amd64 and ci-etcd-unit-test-amd64 runs 6 times a day. Keeping a rolling window of 14 days.
```

Comment:
Could we instead make it time based and not run based?

Reply:
done